### PR TITLE
Feature/371

### LINKED ISSUE #371: As a guardian i would like to resume multiple canceled activities in the weekplan edit screen
**Is your feature request related to a problem? Please describe.**
When in the weekplan edit screen it is possible to cancel multiple activities but not resume them again, this has to be done individually.

unknown

**Describe the solution you'd like**
A button to resume multiple activities.

**Describe alternatives you've considered**
When clicking the cancel button on canceled activities, they could be resumed.

**Additional context**
It is also possible to cancel already canceled activities, without anything happening.

### PR DESCRIPTION
**Pull request by ZohraAmini**
Closes #371 

The Resume button (Genoptag) is created and tested on emulator. The associated flutter test and unit tests are implemented too.

### CODE REVIEW

**Comment by Determinatoro** [comment]
Please use the following checklist, @twaxlol. Feel free to check it off as you go along.

**Code Design** 

- [x] The code is in the right place? (Both in terms of folder structure and class structure)
- [x] The code is not over-engineered. Examples of over-engineering: Implemented behavior for future problems     
- [x] This code could not, to the best of my knowledge, have reused existing code
- [x] The code does not introduce functionality that is unnecessary for solving the problem

**Code Readability** 

- [x] Names are meaningful and self-documenting
- [x] It is understandable, by reading the code, what it does
- [x] The code is simple and readable. i.e. it contains no \"hack\", or obscure solution   

**Code Maintainability**

- [x] Has all the added code been properly commented

Only check one of the items below:

- [ ] The functionality does **not** need to be covered by tests
- [x] The functionality needs to be covered by tests

Only check these if the functionality needs to be covered by tests:

- [x] All functionality is covered by tests
- [x] The tests has sensible names
- [x] By reading the tests you know what they cover
- [x] The tests cover sensible cases. Both happy paths and exception paths
- [x] The tests cover the full functionality. i.e., The tests fails if some of the requested functionality is missing

**Functionality**

For this part, open the app, and test:
- [x] The features that the code claims to implement, are actually implemented
- [x] The code does not look like it has bugs
- [x] All the new screens are reachable through in-app navigation

**Comment by Determinatoro** [comment]
Please use the following checklist, @ggAlexS. Feel free to check it off as you go along.

**Code Design** 

- [x] The code is in the right place? (Both in terms of folder structure and class structure)
- [x] The code is not over-engineered. Examples of over-engineering: Implemented behavior for future problems     
- [x] This code could not, to the best of my knowledge, have reused existing code
- [x] The code does not introduce functionality that is unnecessary for solving the problem

**Code Readability** 

- [x] Names are meaningful and self-documenting
- [x] It is understandable, by reading the code, what it does
- [x] The code is simple and readable. i.e. it contains no \"hack\", or obscure solution   

**Code Maintainability**

- [x] Has all the added code been properly commented

Only check one of the items below:

- [ ] The functionality does **not** need to be covered by tests
- [x] The functionality needs to be covered by tests

Only check these if the functionality needs to be covered by tests:

- [x] All functionality is covered by tests
- [x] The tests has sensible names
- [x] By reading the tests you know what they cover
- [x] The tests cover sensible cases. Both happy paths and exception paths
- [x] The tests cover the full functionality. i.e., The tests fails if some of the requested functionality is missing

**Functionality**

For this part, open the app, and test:
- [x] The features that the code claims to implement, are actually implemented
- [x] The code does not look like it has bugs
- [x] All the new screens are reachable through in-app navigation

File: assets/environments.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "SERVER_HOST": "https://srv.giraf.cs.aau.dk/DEV/API",
+  "SERVER_HOST": "http://10.0.2.2:5000",
```
Comment (twaxlol) [inline assets/environments.json]: This should not be pushed to developer branch.

**Review by twaxlol — CHANGES_REQUESTED** [review]

File: assets/environments.json (hunk 1 above)
Comment (ZohraAmini) [reply assets/environments.json]: @twaxlol that commit is reverted now

**Review by twaxlol — APPROVED** [review]

**Review by ggAlexS — APPROVED** [review]

**Pull request merged**